waku/common/utils/parse_size_units.nim

```
@@ -49,3 +49,28 @@ proc parseCorrectMsgSize*(input: string): uint64 =
   let ret = parseMsgSize(input).valueOr:
     return 0
   return ret
+
+proc parseRelayServiceRatio*(ratio: string): Result[(float, float), string] =
```

```suggestion
proc parseRelayServiceRatio*(ratio: string): Result[(float, float), string] {.raises: [ValueError].}=
```
Let's not hide the exception, so compiler can help using the function properly.